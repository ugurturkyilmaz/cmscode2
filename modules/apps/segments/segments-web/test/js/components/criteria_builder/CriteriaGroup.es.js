/**
 * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
 *
 * This library is free software; you can redistribute it and/or modify it under
 * the terms of the GNU Lesser General Public License as published by the Free
 * Software Foundation; either version 2.1 of the License, or (at your option)
 * any later version.
 *
 * This library is distributed in the hope that it will be useful, but WITHOUT
 * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
 * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
 * details.
 */

import {cleanup, render} from '@testing-library/react';
import React from 'react';
import {wrapInTestContext} from 'react-dnd-test-utils';

import {default as Component} from '../../../../src/main/resources/META-INF/resources/js/components/criteria_builder/CriteriaGroup.es';

const connectDnd = jest.fn((element) => element);

const CriteriaGroup = wrapInTestContext(Component);

describe('CriteriaGroup', () => {
	afterEach(cleanup);

	it('renders', () => {
		const {asFragment} = render(
			<CriteriaGroup connectDragPreview={connectDnd} propertyKey="user" />
		);

		expect(asFragment()).toMatchSnapshot();
	});
});
