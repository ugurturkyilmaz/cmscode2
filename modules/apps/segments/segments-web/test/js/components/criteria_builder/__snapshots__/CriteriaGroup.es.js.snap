// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CriteriaGroup renders 1`] = `
<DocumentFragment>
  <div
    class="criteria-group-item color--user"
  >
    <div
      class="empty-drop-zone-root empty-drop-zone-dashed border-primary rounded"
    >
      <div
        class="drop-zone-target p-5"
      >
        <div
          class="empty-drop-zone-indicator"
        />
      </div>
    </div>
  </div>
</DocumentFragment>
`;
